pkg/mock/engine.go

```diff
@@ -0,0 +1,72 @@
+package mock
```

Comment:
is this related to k8s?
maybe use pkg/mock/k8s/engine.go?

Reply:
@heronrs, is used for both providers. Do you think is it better to move to `pkg/mock/k8s/engine.go` anyway?

Reply:
oh I see, that's ok then